pkg/storegateway/chunk_bytes_pool.go

```diff
@@ -42,9 +48,15 @@ func (p *chunkBytesPool) Get(sz int) (*[]byte, error) {
 	p.requestedBytes.Add(float64(sz))
 	p.returnedBytes.Add(float64(cap(*buffer)))
 
+	p.poolByteStats.WithLabelValues("Get", "requested").Add(float64(sz))
+	p.poolByteStats.WithLabelValues("Get", "len").Add(float64(len(*buffer)))
```

Comment:
What's the usecase to track current length of buf?

Reply:
none! :D completness! haha

Reply:
removed